helix-core/src/main/java/org/apache/helix/manager/zk/zookeeper/ZkClient.java

```diff
@@ -904,13 +904,6 @@ public void process(WatchedEvent event) {
         || event.getType() == Event.EventType.NodeCreated
         || event.getType() == Event.EventType.NodeChildrenChanged;
 
-    if (event.getType() == Event.EventType.NodeDeleted) {
```

Comment:
Do we have a new place to debug log delete event? I didn't see it in this PR. Please confirm. If we don't have, let's keep it here.

Reply:
Feel free to enrich the log message though.

Reply:
updated. The removal is not intended